### PR TITLE
in ipmConfigEpics changed instances of sc1ipm to usrdio

### PR DESCRIPTION
**Pull request by c-tsoi**
## Description

In the script, ipmConfigEpics I changed the sc1ipm to usrdio in the three instances it appears in. This was per Tyler's request to make the names more intuitive. 
## Motivation and Context

https://jira.slac.stanford.edu/browse/LCLSECSD-1131
## How Has This Been Tested?

I ran the command using the change in variable name and got the same results.

### CODE REVIEW

**Review by ZLLentz — APPROVED** [review]
This looks correct to me, based on the description here

**Comment by ZLLentz** [comment]
I just remembered that Tyler is on vacation- I'm adding Silke as a reviewer for awareness and I'll merge if she doesn't object

**Review by silkenelson — APPROVED** [review]
I remember the context and this looks fine to me

**Comment by ZLLentz** [comment]
:+1: this is now merged

File: scripts/ipmConfigEpics [hunk 1]
```diff
@@ -131,11 +131,11 @@ elif [ $hutch == 'cxi' ]; then
 	BASE='CXI:DG3:BMMON'
 	EVR='CXI:DG3:BMMON:EVR'
 	WAVE8='dg3ipm'
-    elif [ $1 == 'sc1ipm' ]; then
+    elif [ $1 == 'usrdio' ]; then
         IOC="IOC:CXI:SC1:BMMON"
         BASE='CXI:SC1:BMMON'
         EVR='CXI:SC1:BMMON:EVR'
```
Comment (ZryletTC) [inline scripts/ipmConfigEpics:137]: @c-tsoi Can you please do a follow-up PR to edit these PVs?

Comment (c-tsoi) [reply scripts/ipmConfigEpics:137]: Just made the PR.

**Pull request merged**